libs/template/src/index.ts

```diff
@@ -1,14 +1 @@
-// EXPERIMENTAL
-// TODO(artur): we're keeping it here until we get a migration script.
-export {
-  UnpatchModule as UnpatchEventsModule,
-  UnpatchDirective,
-} from '@rx-angular/template/unpatch';
-
-// STABLE
-// TODO(artur): we're keeping it here until we get a migration script.
-export { LetModule, LetDirective } from '@rx-angular/template/let';
-
-export { PushModule, PushPipe } from '@rx-angular/template/push';
-
-export { ForModule, RxFor, RxForViewContext } from '@rx-angular/template/for';
+export {};
```

Comment:
why do we have to keep that file anyway?

Reply:
Because an entry file is required for ng-packagr in order to build, if the file is empty then I get the following error:

```
Internal error: failed to get symbol for entrypoint
```